lib/http/header_normalizer.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+module HTTP
+  class HeaderNormalizer
```

Comment:
Perhaps stick this under `http/headers/normalizer.rb`?

Reply:
Right! I moved it over there.